app/backend/src/couchers/servicers/api.py

```diff
@@ -120,6 +122,26 @@
     ParkingDetails.paid_offsite: api_pb2.PARKING_DETAILS_PAID_OFFSITE,
 }
 
+fluency2sql = {
+    api_pb2.LanguageAbility.Fluency.FLUENCY_UNSPECIFIED: None,
```

Comment:
Remove from conversion dict; this API enum value should already have been handled before storing in the database.